aop/src/components/file-uploader/FileUploader.vue

```diff
@@ -133,12 +135,26 @@ export default {
               return;
             }
             for (let pageNumber = 1; pageNumber <= pdfDoc.numPages; pageNumber++) {
+              const errorMessage = `Error reading page ${pageNumber} of the PDF.`;
               try {
-                const imageData = await this.getPage(pdfDoc, pageNumber);
-                const scaledImage = await this.scaleImage(imageData);
-                images.push(scaledImage);
-              } catch {
-                reject(`Error reading page ${pageNumber} of the PDF.`);
+                const imageSource = await this.getPage(pdfDoc, pageNumber);
+                const imageData = await this.getImageData(imageSource);
+                if (imageData.size > MAX_IMAGE_SIZE_BYTES) {
+                  imageData = await this.scaleImage(imageData);
+                } else if (imageData.size < MIN_IMAGE_SIZE_BYTES) {
```

Comment:
Will PDFs with any content be over 20000 bytes? I know this is on the base file uploader in common lib, just wondering if this could cause some weird issues with small PDF pages, say just the last line of an NOA, getting cutout.

Reply:
I tested it with just a small single line of text and it was well over 20kb, plus we know what the forms that they're attempting to upload look like and the ends of them don't contain any relevant information

Reply:
Awesome that sounds great